### ISSUE TITLE
Error: Cannot find module 'node-fetch'

### ISSUE THREAD

**Issue by andykais** [ofrobots/require-so-slow#44]
repost issue #22 here. This is still the case for the latest version of this library. I dont see the problem in the master branch though.

```json
{
  "_from": "require-so-slow",
  "_id": "require-so-slow@1.2.0",
  "_inBundle": false,
  "_integrity": "sha512-5snoIjflBaDjWmQC/7ZKRH59+PUV43jHv0CanX0x9TCp2ifPUpGjjEg8X+x5pR7T8zKkHn8bsFQuP6n10El+nA==",
  "_location": "/require-so-slow",
  "_phantomChildren": {},
  "_requested": {
    "type": "tag",
    "registry": true,
    "raw": "require-so-slow",
    "name": "require-so-slow",
    "escapedName": "require-so-slow",
    "rawSpec": "",
    "saveSpec": null,
    "fetchSpec": "latest"
  },
  "_requiredBy": [
    "#USER"
  ],
  "_resolved": "https://registry.npmjs.org/require-so-slow/-/require-so-slow-1.2.0.tgz",
  "_shasum": "174fbcf1d94ffe7058b2ffdfe5aa7b05ef370ae3",
  "_spec": "require-so-slow",
  "_where": "/Users/andrew/Code/scratchwork/scrape-pages",
  "author": "",
  "bin": {
    "rss": "build/src/cli.js",
    "require-so-slow": "build/src/cli.js"
  },
  "bugs": {
    "url": "https://github.com/ofrobots/require-so-slow/issues"
  },
  "bundleDependencies": false,
  "dependencies": {
    "meow": "^5.0.0",
    "npm-package-arg": "^6.1.0",
    "tmp": "0.0.33",
    "update-notifier": "^2.5.0"
  },
  "deprecated": false,
  "description": "`require`s taking too much time? Profile 'em.",
  "devDependencies": {
    "@types/meow": "^4.0.1",
    "@types/node": "^10.3.2",
    "@types/node-fetch": "^2.1.2",
    "@types/npm-package-arg": "^5.1.0",
    "@types/tape": "^4.2.32",
    "@types/tmp": "0.0.33",
    "@types/update-notifier": "^2.2.0",
    "codecov": "^3.0.2",
    "gts": "^0.8.0",
    "node-fetch": "^2.2.0",
    "nyc": "^12.0.2",
    "source-map-support": "^0.5.6",
    "tape": "^4.9.1",
    "typescript": "~2.9.1"
  },
  "engines": {
    "node": ">=6"
  },
  "files": [
    "LICENSE",
    "package.json",
    "README.md",
    "build/src"
  ],
  "homepage": "https://github.com/ofrobots/require-so-slow#readme",
  "keywords": [
    "require",
    "profiler",
    "timeline",
    "trace_events",
    "trace"
  ],
  "license": "Apache-2.0",
  "main": "build/src/index.js",
  "name": "require-so-slow",
  "repository": {
    "type": "git",
    "url": "git+https://github.com/ofrobots/require-so-slow.git"
  },
  "scripts": {
    "check": "gts check",
    "clean": "gts clean",
    "codecov": "nyc report --reporter=json && codecov -f coverage/*.json",
    "compile": "tsc -p .",
    "fix": "gts fix",
    "posttest": "npm run check",
    "prepare": "npm run compile",
    "pretest": "npm run compile",
    "test": "nyc tape -r source-map-support/register build/test/test*.js"
  },
  "types": "build/src/index.d.ts",
  "version": "1.2.0"
}
```

**Comment by soldair** [comment]
```
$ npx require-so-slow request
npx: installed 126 in 5.703s
Cannot find module 'node-fetch'
Require stack:
- /home/ryaneday/.npm/_npx/142988/lib/node_modules/require-so-slow/build/src/cli.js
```
reproed we should do another release

**Comment by JustinBeckwith** [comment]
I believe we fixed this in the 2.0 release :)
